mix.exs

```diff
@@ -27,12 +28,17 @@ defmodule BambooSmtp.Mixfile do
 
   defp deps do
     [
+      # core
       {:bamboo, "~> 1.6"},
+      {:gen_smtp, "~> 1.0.1"},
+
+      # dev / test
       {:credo, "~> 1.4.1", only: [:dev, :test]},
-      {:earmark, ">= 1.3.2", only: :docs},
       {:excoveralls, "~> 0.13.3", only: :test},
+
+      # doc
+      {:earmark, ">= 1.3.2", only: :docs},
       {:ex_doc, ex_doc_version(), only: :docs},
-      {:gen_smtp, "~> 1.0.1"},
       {:inch_ex, "~> 2.0.0", only: :docs}
```

Comment:
I'm curious about this change. Is this considered a good practice in the Elixir ecosystem? This dependency list is kinda small and is not intended to grow. 🤔

Reply:
@arnaudmorisset Hello, just a matter of preference and easy scanning of dependencies. If you all prefer the original sequence, let me know. I'll amend it. :smiley:

Reply:
Well, I was surprised to see that, but it can't hurt. 🙂

I will put an approve, but I will wait that, at least, one of my coworkers agree too before merging.

Reply:
I like it too 😄